docs/fundamentals/executor/executor-serve.md

```diff
@@ -128,3 +128,99 @@ The above example runs the `DummyHubExecutor` from Executor Hub locally on your
 The Executor you use needs to be already containerized and stored in an accessible registry. We recommend Executor Hub for this.
 ````
 
+(reload-executor)=
+## Reload Executor
+
+While developing your Executor, it can be useful to have the Executor be refreshed from the source code while you are working on it, without needing to restart the complete server.
```

Comment:
We can say:
in order to iterate fast, time consuming jobs (loading a model, downloading weights or files, indexing data,...) are better handled in the Executor constructor or in a module different than the Executor's module. Otherwise, each time you make code changes, reloading the executor will take time

Reply:
I think this would belong somewhere else as it is independent from this feature